README.md

```diff
@@ -43,36 +43,7 @@ Every time you modify Cython files, also run:
 $ python setup.py build_ext -i  # Build binary extensions
 ```
 
-## License (Modified BSD)
-
-Copyright (C) 2011, the scikit-image team
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
-
-1.  Redistributions of source code must retain the above copyright
-    notice, this list of conditions and the following disclaimer.
-2.  Redistributions in binary form must reproduce the above copyright
-    notice, this list of conditions and the following disclaimer in
-    the documentation and/or other materials provided with the
-    distribution.
-3.  Neither the name of skimage nor the names of its contributors may be
-    used to endorse or promote products derived from this software without
-    specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
-IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,
-INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
-(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
-HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
-STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
-IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
-POSSIBILITY OF SUCH DAMAGE.
+## [License](https://github.com/scikit-image/scikit-image/blob/main/LICENSE.txt)
```

Comment:
```suggestion
## License & copyright

See [LICENSE.txt](https://github.com/scikit-image/scikit-image/blob/main/LICENSE.txt).
```

@stefanv, I'm slightly leaning towards not including the link in the header itself. Just seems more expressive and less prone to confuse some poor tool out there. :sweat_smile: But it's not a strong opinion.

Though, I'd prefer it if we included the "& copyright" part in the title regardless of this suggestion. :)